src/video/windows/SDL_windowsrawinput.c

```diff
@@ -78,6 +78,9 @@ static DWORD WINAPI WIN_RawInputThread(LPVOID param)
         devices[count].usUsagePage = USB_USAGEPAGE_GENERIC_DESKTOP;
         devices[count].usUsage = USB_USAGE_GENERIC_KEYBOARD;
         devices[count].dwFlags = 0;
+        if (SDL_GetHintBoolean(SDL_HINT_WINDOWS_RAW_KEYBOARD_NOHOTKEYS, false)) {
+            devices[count].dwFlags |= RIDEV_NOHOTKEYS;
+        }
```

Comment:
I think this is tread-unsafe. Instead of querying the hint here, you should store it as `RawInputThreadData.flags` before starting the thread and simply read it here.